mappings/net/minecraft/RecipeJsonFactory.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_5797 net/minecraft/RecipeJsonFactory
```

Comment:
This feels like it should be in `net.minecraft.data` or a subpackage.

Reply:
Oops, it should be in the same package as the two classes that implement it, `net.minecraft.data.server.recipe`. Fixed in 2d00a36969705a1edecc251736ffe4e0b0f40ff3.